newrelic_lambda_cli/integrations.py

```diff
@@ -342,8 +392,9 @@ def update_log_ingestion_function(input):
 
         # We can't change props during import, so let's set them to their current values
         lambda_client = input.session.client("lambda")
+        unique_log_ingestion_name = get_unique_newrelic_log_ingestion_name(input.session)
```

Comment:
This looks like it's grabbing settings from the existing log ingestion function, if the existing log ingestion function is the old function name we should grab them from the old name here and again warn the user about having the old log ingestion function in their region. Something like the following:
```suggestion
        unique_log_ingestion_name = get_unique_newrelic_log_ingestion_name(input.session)
        old_props = {}
        try:
            old_props = lambda_client.get_function_configuration(
                FunctionName=unique_log_ingestion_name
            )
        except Lambda.Client.exceptions.ResourceNotFoundException:
            old_props = lambda_client.get_function_configuration(
                FunctionName="newrelic-log-ingestion"
            )
            warning("It looks like an old log ingestion function is present in this region. Pulling the settings from that. Consider manually deleting this as it is no longer used and has been replaced by a log ingestion function specific to the stack.")
```